utils/protocolImport.tsx

```diff
@@ -21,14 +21,37 @@ export const getProtocolJson = async (protocolZip: Zip) => {
  * return them as a collection of ProtocolAsset objects, which includes useful
  * metadata about the asset.
  */
+
+type ProtocolAsset = {
+  assetId: string;
+  name: string;
+  type: string;
+}
+
+type FileAsset = {
+  file: File;
+} & ProtocolAsset
+
+type ApiKeyAsset = {
+  value: string;
+  key: string;
+  size: number;
+  url: string;
+} & ProtocolAsset
```

Comment:
Aren't these types defined anywhere else? It feels like these should live in either the protocol schema, the prisma schema, or shared-consts.

Reply:
resolved [28ede8c](https://github.com/complexdatacollective/Fresco/pull/236/commits/28ede8cd31511e81182a96c7c536f081054252d7)